src/main/java/nl/fuchsia/exceptionhandlers/UniekVeldException.java

```diff
@@ -0,0 +1,11 @@
+package nl.fuchsia.exceptionhandlers;
+
+public class UniekVeldException extends RuntimeException {
+    public UniekVeldException(String msg) {
+        super(msg);
+    }
+
+    public UniekVeldException(String msg, Throwable cause) {
```

Comment:
Deze methode is niet nodig bij een subclasse van een RuntimeException. Deze stond er nog toen deze ge-extend was naar een Exception.